docs/tutorials/integrations/nginx/tls/tls.conf

```diff
@@ -0,0 +1,39 @@
+[req]
+default_bits = 2048
+prompt = no
+default_md = sha256
+req_extensions = req_ext
+distinguished_name = dn
+x509_extensions = v3_ca # The extentions to add to the self signed cert
+req_extensions  = v3_req
+x509_extensions = usr_cert
```

Comment:
I'm confused about these extensions. If you want the same extensions for req and x509, you should point them to the same section instead of duplicating it. Additionally you reference `req_ext` which does not exist.

Reply:
I tried removing this duplication but ran into certificate errors. See comment below.